bundles/core/src/main/java/com/adobe/cq/commerce/core/components/internal/models/v1/product/ProductImpl.java

```diff
@@ -244,6 +246,14 @@ public Boolean isBundleProduct() {
         return isBundleProduct;
     }
 
+    @Override
+    public Boolean isGiftCardProduct() {
```

Comment:
Can you also add a unit test to cover that?

Reply:
Added now